### ISSUE TITLE
Typo in require path #326 Still a problem in 0.0.42

### ISSUE THREAD

**Issue by rodrigobnn** [driftyco/ionic-app-scripts#353]
Typo in require path #326 is still a problem for linux users in app-scripts 0.0.42

**Comment by danbucholtz** [comment]
This will be published today.

Thanks,
Dan

**Comment by danbucholtz** [comment]
This is resolved. Please install `npm install @ionic/app-scripts@latest`.

Thanks,
Dan

**Comment by rodrigobnn** [comment]
Tks @danbucholtz

**Comment by NoelLauigan** [comment]
Hi @danbucholtz .

Thanksfor the support but how can I see the sample app in my browser I am using cloud9.
Like before we can choose if local host or the private ip 172.31... but now only it says: 
[21:36:23]  dev server running: http://localhost:8100/ 

Have I missed something. Thanks.
=================================================
[21:36:10]  ionic-app-scripts 0.0.43 
[21:36:10]  watch started ... 
[21:36:10]  build dev started ... 
[21:36:10]  clean started ... 
[21:36:10]  clean finished in 4 ms 
[21:36:10]  copy started ... 
[21:36:10]  transpile started ... 
[21:36:13]  transpile finished in 2.67 s 
[21:36:13]  webpack started ... 
[21:36:13]  copy finished in 2.80 s 
[21:36:19]  webpack finished in 6.42 s 
[21:36:19]  sass started ... 
[21:36:23]  sass finished in 4.06 s 
[21:36:23]  build dev finished in 13.20 s 
[21:36:23]  watch ready in 13.27 s 
[21:36:23]  dev server running: http://localhost:8100/ 
=============================================

**Comment by danbucholtz** [comment]
@NoelLauigan,

Please create a separate issue for that as it's unrelated.

Thanks,
Dan
